code/obj/machinery/manufacturer.dm

```diff
@@ -1651,6 +1689,23 @@
 		</tr>
 			"}
 
+		if (src.fabdisk)
+			dat += {"
+		<tr><th colspan='2'>Manufacturer disk</th></tr>
+			"}
+
+			dat += {"
+		<tr><td colspan='2'><a href='?src=\ref[src];ejectfabdisk=\ref[src.beaker]' class='buttonlink'>&#9167;</a> [src.fabdisk.name]</td></tr>
```

Comment:
`src.beaker` stuff looks like copy-pasta.

Reply:
Was initially copy pasted from beaker code for it since I dont know HTML, fixed it.